internal/circle/client_test.go

```diff
@@ -290,12 +309,14 @@ func Test_Client_GetWalletByID(t *testing.T) {
 	})
 }
 
-func newClientWithMock(t *testing.T) (Client, *httpclientMocks.HttpClientMock) {
+func newClientWithMock(t *testing.T) (Client, *httpclientMocks.HttpClientMock, *tenant.TenantManagerMock) {
 	httpClientMock := httpclientMocks.NewHttpClientMock(t)
+	tntManagerMock := &tenant.TenantManagerMock{}
```

Comment:
💡  If you use `tenant.NewTenantManagerMock(t)`, the AssertExpectations(t) should be called automatically.